envs/environment_py37_iris30.yml

```diff
@@ -21,10 +21,10 @@ dependencies:
   - scipy=1.6
   - sigtools
   - sphinx
-  - statsmodels
   # Optional
   - numba
   - pysteps=1.4.1
+  - statsmodels
```

Comment:
I think I need this explaining. The statsmodel dependency has moved from "Required" to "Optional", but the PR removes the optional status of this module, which I think makes it "Required".

Reply:
My previous understanding of "Required" and "Optional" was just like you suggest, which is why I made this PR: https://github.com/metoppv/improver/pull/1556. However, the preference, is to treat statsmodels more like pysteps, so even though it is required for EMOS, it is optional for the improver codebase, as someone might want to use improver, but not the EMOS component.